services/route_manager/src/main/resources/application.properties

```diff
@@ -18,6 +18,7 @@ microservices.subnet.service.url=http://localhost:9002
 microservices.vpc.service.url=http://localhost:9001
 microservices.dpm.service.url=http://localhost:9010
 microservices.port.service.url=http://localhost:9006
+microservices.dataplane.service.url=http://localhost:9010/network-configuration
```

Comment:
All the external service urls need to add to Kubernetes yaml files as well so that K8s deployment will take effect. 

https://github.com/futurewei-cloud/alcor/blob/559f2bd90ec4273dfef10024e657739bd4fb6c15/kubernetes/services/dpm_manager.yaml#L50